OWNERS

```diff
@@ -0,0 +1,5 @@
+assignees:
+- brendandburns
```

Comment:
- brendan, probably. I think this should include the people actually maintaining the docs repo: jared, kelsey, phillip, janetkuo. Here and throughout.